ui/src/kapacitor/components/RuleHeader.js

```diff
@@ -110,31 +110,31 @@ export const RuleHeader = React.createClass({
           onKeyDown={e => this.handleEditName(e, rule)}
           onBlur={() => this.handleEditNameBlur(rule)}
           placeholder="Name your rule"
+          spellCheck={false}
+          autoComplete={false}
         />
-      : <h1
-          className="page-header__title page-header--editable kapacitor-theme"
-          onClick={this.toggleEditName}
-          data-for="rename-kapacitor-tooltip"
-          data-tip="Click to Rename"
-        >
-          {rule.name}
-          <span className="icon pencil" />
-          <ReactTooltip
-            id="rename-kapacitor-tooltip"
-            delayShow={200}
-            effect="solid"
-            html={true}
-            offset={{top: 2}}
-            place="bottom"
-            class="influx-tooltip kapacitor-tooltip place-bottom"
-          />
-        </h1>
+      : <div className="page-header__left">
+          <h1
+            className="page-header__title page-header--editable kapacitor-theme"
```

Comment:
I concur. Been meaning to change these for a while. For a brief time I was using `__` when I should have been using `--` and then quickly switched to the correct pattern afterwards, and these styles are the byproduct. Also it's super frustrating that text editors select each differently when both are clearly separators